cmd/image-builder/main.go

```diff
@@ -97,6 +99,37 @@ func ostreeImageOptions(cmd *cobra.Command) (*ostree.ImageOptions, error) {
 	}, nil
 }
 
+type registrations struct {
+	Redhat struct {
```

Comment:
We could of course add one more level here if we wanted like
```json
{
 "registrations": {
  "redhat": {...},
}
```
no strong opinion here.

Reply:
I suppose.  I'm all for the idea in general, but didn't think about the structure for too long.  We should probably consider everything we might want to add going forward before finalising.

Reply:
Absolutely, this is the place where we can put AAP as well.

Reply:
I don't particularly see much value in having an extra level of `registrations`, do we want to put things other than registrations in the registrations file? Ansible Automation Platform could be its own top-level key, or it could be a key under Red Hat.